lib/main_common.pm

```diff
@@ -302,6 +302,17 @@ sub is_leanos {
     return 0;
 }
 
+sub is_sle12sp2_using_system_role {
```

Comment:
Seems that this one is not used anywhere now, is it?

Reply:
yes, I refactored and forgot to include it in the code after the revert. thx